### PR TITLE
refactor: Use type-safe time in txorphanage

### PR DESCRIPTION
**Pull request by maflcko**
This allows to remove manual conversions like multiplication by `60`, and uses a type-safe type instead of a raw `int64_t`.

### CODE REVIEW

**Comment by stickies-v** [comment]
Concept ACK

File: src/txorphanage.cpp [hunk 1]
```diff
@@ -118,12 +119,12 @@ void TxOrphanage::LimitOrphans(unsigned int max_orphans, FastRandomContext& rng)
     LOCK(m_mutex);
 
     unsigned int nEvicted = 0;
-    static int64_t nNextSweep;
-    int64_t nNow = GetTime();
+    static NodeSeconds nNextSweep;
```
Comment (dergoegge) [inline src/txorphanage.cpp:122]: Out of scope for this PR but this global is probably the source of instability in the txorphan harness.

**Review by dergoegge — APPROVED** [review]
Code review ACK fa6d4891c7815025ab1cd58d0906466af31bb648

**Review by brunoerg — APPROVED** [review]
utACK fa6d4891c7815025ab1cd58d0906466af31bb648

**Pull request merged**